### PR TITLE
Updated Installation Instructions

### PR DESCRIPTION
**Pull request by maingotj**
Updated Installation Instructions for the forge mod loader by adding a link to the download page and additional instructions on how to configure the download. Added clarification for where to move the VulkanMod.jar and how to do it. Additionally added multiple formatting tweaks such as left aligning the bottom and deleting a duplicate download link.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,15 +16,46 @@ Demostration Video:
 
 [![ko-fi](https://ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/V7V7CHHJV)
 
+</div>
+
+
 ## Installation
+Proceed through all of these steps to correctly install the Vulcan mod to your Windows device
```
Comment (iMonZ) [inline README.md:23]:
```suggestion
Proceed through all of these steps to correctly install the Vulkan mod to your device
```

1. Vulkan is wrongly written
2. This mod is not only for Windows

**Review by iMonZ — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -16,15 +16,46 @@ Demostration Video:
...
 ### Download
 [![CurseForge](https://cf.way2muchnoise.eu/title/635429_Get_Mod.svg?badge_style=flat)](https://www.curseforge.com/minecraft/mc-mods/vulkanmod)
 
-[![CurseForge](https://cf.way2muchnoise.eu/full_635429_downloads.svg?badge_style=flat)](https://www.curseforge.com/minecraft/mc-mods/vulkanmod)
+### Fabric mod loader
+Fabric mod loader is needed to run the Vulcan Mod
+
+1. To download Fabric go to the [fabric installation page](https://fabricmc.net/use/installer/)
+
+1. Once on the page click the blue "download for Windows" button
```
Comment (iMonZ) [inline README.md:33]: Again Same issue

**Comment by iMonZ** [comment]
Thanks but this tutorial is only for MS-Windows.
So how about we rebrand it like that since other OSes having different installation methods.

**Comment by xCollateral** [comment]
Since there are no meaningful changes and we are  working on #383, I'll close this PR.

**Pull request closed without merging**